chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -290,6 +290,7 @@ def find_node_boot_mac_addresses(node, admin_data_net)
           append << "ifcfg=dhcp4 netwait=60"
           append << "squash=0" # workaround bsc#962397
           append << "autoupgrade=1" if mnode[:state] == "os-upgrading"
+          append << "self_update"
```

Comment:
Also, if the URL contains `<ADMINWEB>`, I think we could do some quick check with `File.exists?` so that if we use a default with `<ADMINWEB>` but the repo is not mirrored, we just don't enable the update feature.